Store/StoreBraintreePaymentProvider.php

```diff
@@ -156,6 +156,8 @@ public function pay(
 			throw $this->generateExceptionFromResponse($response);
 		}
 
+		throw new StorePaymentBraintreeValidationException('Nick test');
```

Comment:
For testing

Reply:
needs to be reverted before being merged

Reply:
I turned on the same CVV checking on the sandbox, and I think we can now test without this code by using the CVV `200`.

Reply:
we should make a Cypress test for that